src/audio_sdl_mixer.h

```diff
@@ -73,6 +73,10 @@ class SdlMixerAudio : public AudioInterface {
 	bool played_once = false;
 
 	struct sound_data {
+		sound_data() = default;
+		sound_data(const sound_data&) = delete;
+		sound_data& operator=(sound_data&&) = default;
```

Comment:
Those are correct, but you also need to delete copy assignment and default move construction